packages/docs/modules/page-config/blocks/api/common-description.ts

```diff
@@ -17,7 +17,7 @@ export default
   props: {
     align: "Customizes horizontal position of component (flex based). Available values are strings: 'left', 'center', 'right', 'between', 'around', 'stretch'.",
     id: "Applies `id` to internal input component. Useful for native forms.",
-    name: "Applies `name` to internal input component. Useful for native forms.",
+    name: "`name` represents icon name which will be used by [Icons Config](https://ui.vuestic.dev/services/icons-config).",
```

Comment:
I think it must be commonly used as name in terms of form, but for Icon component only this must have different description.

Reply:
@m0ksem Okay, I'd be happy to make those changes, but first need a bit more clarification. Would the text changes need to be made in the _'api-description.ts'_ file found by following the file path: _packages >docs >page-config > ui-elements > icon > api-description.ts_ instead or maybe another file? The screenshot at the top of issue #3698 unfortunately didn't make it very clear what specific files need editing. Thanks for your help clarifying!

Reply:
HI, @deverestHood. Yes. The name should be changed only for page-config/icon in its own api description. Because right now it changes every `name` prop. It must be done for icon only. Hope it helps!

Reply:
@m0ksem Hello! I've added the name prop for page-config/icon/api-description. Thanks for your help!